apps/dialog/src/routes/dialog/wallet_connect.tsx

```diff
@@ -48,28 +49,52 @@ function RouteComponent() {
 
       const params = request.params ?? []
 
-      return Actions.respond(porto, {
-        ...request,
-        params: [
-          {
-            ...params[0],
-            capabilities: {
-              ...params[0]?.capabilities,
-              createAccount: email
-                ? {
-                    ...(typeof params[0]?.capabilities?.createAccount ===
-                    'object'
-                      ? params[0]?.capabilities?.createAccount
-                      : {}),
-                    label: email,
-                  }
-                : params[0]?.capabilities?.createAccount || !signIn,
-              email: Boolean(email),
-              selectAccount,
+      return Actions.respond(
+        porto,
+        {
+          ...request,
+          params: [
+            {
+              ...params[0],
+              capabilities: {
+                ...params[0]?.capabilities,
+                createAccount: email
+                  ? {
+                      ...(typeof params[0]?.capabilities?.createAccount ===
+                      'object'
+                        ? params[0]?.capabilities?.createAccount
+                        : {}),
+                      label: email,
+                    }
+                  : params[0]?.capabilities?.createAccount || !signIn,
+                email: Boolean(email),
+                selectAccount,
+              },
             },
+          ],
+        },
+        {
+          onError: (e) => {
+            if (
+              e?.message?.includes("Invalid 'sameOriginWithAncestors' value")
+            ) {
+              Dialog.store.setState({
+                error: {
+                  action: 'retry-in-popup',
+                  message:
+                    'Your browser doesn’t support passkey creation in the current context.',
+                  name: 'CREDENTIAL_CREATION_FAILED',
+                  secondaryMessage:
+                    'Please try again in a popup window for better compatibility.',
+                  title: 'Passkey Creation Not Supported',
```

Comment:
Let's switch to sentence case

```suggestion
                  title: 'Passkey creation not supported',
```